src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx

```diff
@@ -227,7 +227,7 @@ export default class Oncoprinter extends React.Component<
                 this.props.store.driverAnnotationSettings.customBinary = s;
             }),
             onSelectHidePutativePassengers: (s: boolean) => {
-                this.props.store.driverAnnotationSettings.excludeVUS = s;
+                this.props.store.driverAnnotationSettings.includeVUS = !s;
```

Comment:
i wonder if we should change "PutativePassengers" to VUS?  seems like two different words for the same thing

Reply:
Yes, done.